### PR TITLE
Moving confusing docker param from main setup page

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Moves docker parameter to the dedicated tracing/setup/docker page

### Motivation
A customer was confused

### CODE REVIEW

File: content/tracing/setup/docker.md [hunk 1]
```diff
@@ -16,7 +16,25 @@ Enable the [datadog-trace-agent][1] in the `datadog/agent` container by passing
...
+| `DD_HOSTNAME`              | Set manually the Agent hostname.                                                              |
+| `DD_BIND_HOST`             | Set the StatsD & receiver hostname.                                                           |
+| `DD_RECEIVER_PORT`         | Port that the Datadog Agent's trace receiver listen on. Default value is `8126`.              |
+| `DD_DOGSTATSD_PORT`        | Set the DogStatsD port.                                                                       |
+| `DD_APM_NON_LOCAL_TRAFFIC` | Allow non-local traffic when [tracing from other containers](#tracing-from-other-containers). |
+| `DD_IGNORE_RESOURCE`       | A comma-separated list of resources that the Agent should ignore.                             |
+| `DD_LOG_LEVEL`             | Set the logging level.                                                                        |
+| `DD_APM_ANALYZED_SPANS`    | Configure the spans to analyze for transactions.                                              |
+| `DD_CONNECTION_LIMIT`      | Set the Limit of unique connections .                                                         |
```
Comment (gbbr) [inline content/tracing/setup/docker.md]: You might want to lower case "_limit_" and remove the space before the period at the end of the sentence.

File: content/tracing/setup/docker.md [hunk 2]
```diff
@@ -16,7 +16,25 @@ Enable the [datadog-trace-agent][1] in the `datadog/agent` container by passing
...
+| `DD_APM_DD_URL`            | Datadog API endpoint where traces are sent.                                                   |
+| `DD_PROXY_HTTPS`           | Set up the URL for the proxy to use.                                                          |
+| `DD_HOSTNAME`              | Set manually the Agent hostname.                                                              |
+| `DD_BIND_HOST`             | Set the StatsD & receiver hostname.                                                           |
+| `DD_RECEIVER_PORT`         | Port that the Datadog Agent's trace receiver listen on. Default value is `8126`.              |
+| `DD_DOGSTATSD_PORT`        | Set the DogStatsD port.                                                                       |
+| `DD_APM_NON_LOCAL_TRAFFIC` | Allow non-local traffic when [tracing from other containers](#tracing-from-other-containers). |
+| `DD_IGNORE_RESOURCE`       | A comma-separated list of resources that the Agent should ignore.                             |
+| `DD_LOG_LEVEL`             | Set the logging level.                                                                        |
```
Comment (gbbr) [inline content/tracing/setup/docker.md]: Make a parenthesis and give an example of possible values. You can see them [here](https://github.com/DataDog/datadog-trace-agent/blob/cf995d193c5b6947812d520d15430a2561728cdf/vendor/github.com/cihub/seelog/common_loglevel.go#L43-L49)

File: content/tracing/setup/_index.md [hunk 3]
```diff
@@ -52,23 +52,23 @@ apm_config:
...
+| `receiver_port`           | number      | Port that the Datadog Agent's trace receiver listen on. Default value is `8126`.                                                                                   |
+| `apm_non_local_traffic`   | boolean     | Allows the Agent to receive outside connections. It then listen on all interfaces.                                                                                 |
+| `max_memory`              | float       | Maximum memory that the Agent is allowed to occupy. When this is exceeded the process is killed.                                                                   |
+| `max_cpu_percent`         | float       | Maximum CPU percentage that the Agent should use. The Agent automatically adjusts its pre-sampling rate to stay below this number.                                 |
+| `max_connections`         | number      | Maximum number of network connections that the Agent is allowed to use. When this is exceeded the process is killed.                                               |
```
Comment (LotharSee) [inline content/tracing/setup/_index.md]: This is an option we deprecated, we'd like to remove it from the public documentation.

Comment (LotharSee) [reply content/tracing/setup/_index.md]: Same with `DD_CONNECTION_LIMIT`

Comment (gbbr) [reply content/tracing/setup/_index.md]: @l0k0ms can we remove them?

Comment (gbbr) [reply content/tracing/setup/_index.md]: Created https://github.com/DataDog/datadog-trace-agent/pull/469 to ensure this is clear moving forward

Comment (l0k0ms) [reply content/tracing/setup/_index.md]: https://github.com/DataDog/documentation/pull/3123

**Pull request merged**